### ISSUE TITLE
Include recommendations framework and tradeoffs 

### ISSUE THREAD

**Issue by abarciauskas-bgse** [NASA-IMPACT/zarr-visualization-report#16]
Here's a proposed outline for the set of recommendations we want to provide by the end of September:
- If you're going to use a tile server...
    - if you can't change your data, here's what you can do to visualize it and what the experience will be
    - if you can change your data, here's what you should do and how much it will cost
- If you're going to use the dynamic client...
   - if you can't change your data, here's what you can do to visualize it and what the experience will be
   - if you can change your data, here's what you should do and how much it will cost

We will also include a set of tradeoffs.

**Comment by maxrjones** [comment]
> - If you're going to use the dynamic client...
> -     if you can't change your data, here's what you can do to visualize it and what the experience will be

For the dynamic client, I think a section documenting the necessary development steps that would unblock visualizing data that can't be changed is more applicable. This could include an outline of how the approach could be extended to render data that doesn't include pyramids as well as a roadmap for integrating pyramids into "analysis ready" Zarr stores (i.e., multiscales ZEP) (cc @katamartin for any additional thoughts).

**Comment by abarciauskas-bgse** [comment]
we added tradeoffs in https://nasa-impact.github.io/zarr-visualization-report/approaches/tradeoffs.html

@maxrjones should we add a new ticket to document the necessary development steps to unblock visualizing data or are you happy with the content as-is?

**Comment by maxrjones** [comment]
@maxrjones should we add a new ticket to document the necessary development steps to unblock visualizing data or are you happy with the content as-is?

I think the content is good as-is!
